experiments/AMIP/modular/components/atmosphere/climaatmos_init.jl

```diff
@@ -1,9 +1,16 @@
 # atmos_init: for ClimaAtmos pre-AMIP interface
 import ClimaAtmos
 using ClimaAtmos: RRTMGPI
-import ClimaCoupler.FluxCalculator: compute_atmos_turbulent_fluxes!, calculate_surface_air_density
+import ClimaAtmos: CT1, CT2, CT12, CT3, C3, C12, unit_basis_vector_data, ⊗
```

Comment:
If we want to qualify these, we could `import ClimaAtmos as CA` and call them as `CA.CT...`, but I'm not sure that's much better

Reply:
Yeah, also, this will live in Atmos soon, so I wouldn't worry too much.